.pyup.yml

```diff
@@ -1,6 +1,4 @@
-update: insecure
+schedule: "every month"
```

Comment:
Does removing "update: insecure" means we will get *all* updates? This is quite spammy imho

Reply:
Yes, but only once a month. Our open pull requests suggest we've been getting some anyway (though maybe they're insecure). I'm happy to review these when they come in, and perhaps switch back to only insecure updates in the future.